packages/libs/bootstrap-lib/package.json

```diff
@@ -42,18 +40,18 @@
   "dependencies": {
     "@kadena/cryptography-utils": "workspace:*",
     "@kadena/pactjs": "workspace:*",
-    "@kadena/types": "workspace:*",
-    "@types/isomorphic-fetch": "~0.0.36",
     "cross-fetch": "~3.1.5",
     "node-fetch": "~2.6.2"
   },
   "devDependencies": {
     "@kadena-dev/eslint-config": "workspace:*",
     "@kadena-dev/heft-rig": "workspace:*",
     "@kadena-dev/markdown": "workspace:*",
+    "@kadena/types": "workspace:*",
     "@rushstack/eslint-config": "~3.3.0",
     "@rushstack/heft": "~0.50.6",
     "@types/heft-jest": "~1.0.3",
+    "@types/isomorphic-fetch": "~0.0.36",
```

Comment:
Changes in dependencies of this `packages.json` are examples of what synpack detected.